docs/courses/design/brainstorming-analogies.md

```diff
@@ -0,0 +1,69 @@
+
+> To compare bulk and single-cell RNA-seq, I like to use the analogy with the milkshake versus pieces of fruits (idea from Shalek and Regev in [a talk](https://image.slidesharecdn.com/whyscientistanalyzesinglecells-161116191653/95/why-scientist-analyze-single-cells-4-638.jpg?cb=1501073560))
+
+From a course on forecasting product demand. Again, this analogy is likely be intuitive to most people.
+
+> Signal and noise - It's like trying to hear someone across a crowded room. Remove the noise and you can understand easily what they are telling you.
+
+There's a [funny image](https://effectsizefaq.com/2010/05/31/i-always-get-confused-about-type-i-and-ii-errors-can-you-show-me-something-to-help-me-remember-the-difference) analogy used to explain Type I and II statistical errors.
+
+An analogy from Albert Kim:
+
+> R packages are like apps, and CRAN is like the App Store.
+
+[IBM](https://www.ibm.com/support/knowledgecenter/en/SS3RA7_16.0.0/com.ibm.spss.modeler.help/clementine/jython/clementine/svmnodeslots.htm) heuristic on machine learning model choice:
+
+> Support Vector Machines work well with wide data sets, such as those with a very large number of input fields.
+
+Computer science is full of heuristics on good coding practice. From The Elements of Programming Style:
+
+> debugging is twice as hard as writing a program in the first place
+
+From Testing R Code:
+
+> Most R functions should be 7 lines of code or less.
+
+[Siddiqui 2013](https://pdfs.semanticscholar.org/fa7a/1c2e306b2aa2a4e9f8b15de7075246e1e0ba.pdf) provides some heuristics on minimum sample sizes for various statistical techniques
+
+> For multiple regression analyses the desired level is between 15 to 20 observations for each predictor variable.
+
+
+## FAQs
+
+### What is the difference between an analogy and a heuristic?
+
+Analogies are comparisons between similar things. In this case, you want an analogy that compares the technical concept in your course, with a concept from every-day life.
+
+Heuristics are sometimes called "rules of thumb". They are practical rules that work a lot of the time.
+
+### This is hard. Can I skip it?
+
+You are not alone. Many instructors struggle with this. If you are stuck, come back to this question after you've written the rest of the spec. You should try and write something here eventually, since it's important to try and find non-technical explanations for the concepts in your course.
+
+
+## Good ideas
+
+### What don't you write down?
+
+Many heuristics are the things that you do in your day job, but don't write down because you just know them. Many DataCamp students don't know these things yet, so now is them time to write them!
+
+
+## Common problems and their solutions
+
+### I can't think of anything
+
+- Try reading [Statistical Teaching: Bag of Tricks](http://www.stat.columbia.edu/%7Egelman/bag-of-tricks/) for inspiration.
```

Comment:
Maybe something "Try scanning over" instead of "Try reading"? Seems like quite the daunting task to ask instructors to read a book to help with brainstorming.

Reply:
fixed